src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -616,23 +635,19 @@ private void prepareStudentExamForConduction(HttpServletRequest request, User cu
 
                 // Set up new participations for the Exercises and set initialisationDate to the startedDate
                 studentExamService.setUpTestExamExerciseParticipationsAndSubmissions(studentExam, startedDate);
-
-                // Mark the student exam as started and save it
-                studentExam.setStarted(true);
-                studentExam.setStartedDate(startedDate);
-                studentExamRepository.save(studentExam);
             }
         }
-        else {
-            // Mark the student exam as started and save it
+
+        if (!studentExam.isStarted() || studentExam.getStartedDate() == null) {
+            // Mark the student exam as started with now as the start date if it was not started before
+            var startDate = studentExam.getStartedDate() != null ? studentExam.getStartedDate() : ZonedDateTime.now();
             studentExam.setStarted(true);
```

Comment:
Probably unrelated to this PR, but logically speaking (from a point of view as someone who's still getting to know the codebase), wouldn't it be better to remove those setters and add a function like `studentExam.startExam(ZonedDateTime startedDate)` that sets both started to true and the start date to the given date?

Reply:
Yes, this would make sense

Reply:
ok